server/engine/tests/test_write_opensim.py

```diff
@@ -0,0 +1,26 @@
+import nimblephysics as nimble
+import unittest
+import os
+from inspect import getsourcefile
+import shutil
+from outputs.opensim_writer import write_opensim_results
+import tempfile
+
+TESTS_PATH = os.path.dirname(getsourcefile(lambda:0))
+TEST_DATA_PATH = os.path.join(TESTS_PATH, 'data')
+
+
+class TestWriteOpenSim(unittest.TestCase):
```

Comment:
Similar to combining the "dynamics pass" unittests: consider combining `TestWriteOpenSim` and `TestWriteWebResults` into one file (e.g., `test_writing.py` or `test_write_files.py`).